### DISCUSSION TITLE
Is the button public?

### DISCUSSION THREAD

**Discussion by contigen** [open-sauced-craftwork/browser-extension#36] [Q&A] [answered]
Hello people. The button that's to be displayed, is it a public one, or only the authenticated user gets to see it?

**Comment by anush008** [comment, marked as answer]
It is to be displayed to all the users regardless of their auth status.

**Comment by Rishav1707** [comment]
> Hello people. The button that's to be displayed, is it a public one, or only the authenticated user gets to see it?

You can add features like if the user is validated (account must be there in open sauced website ) then you show the View button else you can show the join open sauced button.

**Comment by Rishav1707** [comment]
> Hello people. The button that's to be displayed, is it a public one, or only the authenticated user gets to see it?

Those user who do not have account on open sauced you can make him join

**Reply by akash-singh8** [reply]
That's a great feature! Can you please tell how exactly we can make users join OpenSauced?

**Reply by Rishav1707** [reply]
Render the `Join open sauced` button(when user not validated) and insert the `onClick` attributes to the button which open the authentication page of the Open sauced website.

**Reply by akash-singh8** [reply]
Sounds great! Can you please share the link for the authentication page of the OpenSauced?

**Reply by Rishav1707** [reply]
https://insights.opensauced.pizza/start

**Reply by akash-singh8** [reply]
Thankyou bro.
